R/promptAndRun.R

```diff
@@ -15,27 +15,28 @@
 #' }
 #'
 promptAndRun <- function(mydir = ".", user = NULL, daysback = 3) {
-  if (is.null(user)) user <- Sys.info()[["user"]]
-  if (user == "") user <- Sys.info()[["user"]]
+  if (is.null(user) || user == "") user <- Sys.info()[["user"]]
   if (daysback == "") daysback <- 3
-
+  colors <- ! grepl("-.*b.*", mydir)
+  if (isFALSE(colors)) mydir <- gsub("b", "", mydir)
   if (mydir == ".") {
-    loopRuns(".", user = user)
+    loopRuns(".", user = user, colors = colors)
   } else if (mydir == "") {
     if (sum(file.exists(c("full.gms", "log.txt", "config.Rdata", "prepare_and_run.R", "prepareAndRun.R"))) >= 4) {
-      loopRuns(".", user = user)
+      loopRuns(".", user = user, colors = colors)
     } else {
-      dirs <- c(".", list.dirs(".", recursive = FALSE))
```

Comment:
If you run `rs2` in the REMIND folder, search for runs in `output` subfolder instead of presenting nonsense to the user